### DISCUSSION TITLE
I just upgraded my GPU, i should still use same arguments or i should change something?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [Panchovix/stable-diffusion-webui-reForge#124] [Q&A]
RTX 4070 Ti 16Gb Super Asus Tuf Gaming Oc

I have this arguments in the **webui-user.bat**:

set COMMANDLINE_ARGS=--xformers --api --disable-nan-check --medvram-sdxl

**PD:** Talking about **SDXL**, Hyper, Lighting models, not FLUX.

**Comment by PaulZ0** [comment]
You can remove --medvram-sdxl as you have enough VRAM for SDXL. You can also consider --always-gpu if you don't have any other programs with significant VRAM usage running along with reForge.

**Reply by ZeroCool22** [reply]
> You can remove --medvram-sdxl as you have enough VRAM for SDXL. You can also consider --always-gpu if you don't have any other programs with significant VRAM usage running along with reForge.

Thx you!

I also read something about TensorRT that will speed up Stable Diffusion for the RTX cards but i don't know if mess with that and if will cause more errors than advantages, i read Loras must be converted or something...

https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT

**Reply by ZeroCool22** [reply]
> --always-gpu

launch.py: error: unrecognized arguments: --always-gpu

**Reply by brknsoul** [reply]
You can also remove --xformers. The 4070 is powerful enough to take advantage of the speed increases offered by Scale-Dot-Product (sdp).
